packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -84,6 +141,9 @@ export default {
     sharedWithLabel() {
       return this.$gettext('Shared with')
     },
+    spaceMemberLabel() {
+      return this.$gettext('Space Members')
+    },
 
```

Comment:
Still have no clue why those simple translations, require an additional computed, I would purge em

Reply:
Also, should it not be `Space members` for consistency?

Reply:
> Still have no clue why those simple translations, require an additional computed, I would purge em

Didn't want to change existing stuff, and it surely doesn't hurt 😄  But I can do so with the next iteration.

> Also, should it not be Space members for consistency?

@tbsbdr What do you think, `Space members` or `Space Members`? I'd also go for the first here.

Reply:
Adjusted the wording to `Space members` 👍  I also took a look at the translation -> it is used more than once, hence the computed.